### PR TITLE
Fix configuration data for keyboard pom_keyboards/tnln95

### PR DESCRIPTION
**Pull request by NMHoang05**
I checked my keyboard layout on configurator and it doesn't work as intended.

## Description

I update the info file.
~~I commented out layout "all" to avoid confusion in the configurator (it appears despite not declared in the info file).~~
I add layout all to info file
I rearrange the layout "ansi".

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [x] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/pom_keyboards/tnln95/tnln95.h [hunk 1]
```diff
@@ -21,6 +21,7 @@
 // The first section contains all of the arguments
 // The second converts the arguments into a two-dimensional array
 
+/* commented this out to prevent confusion in the configurator
```
Comment (fauxpark) [inline keyboards/pom_keyboards/tnln95/tnln95.h:24]: Don't do this, instead add `LAYOUT_all` to the info.json.

Comment (NMHoang05) [reply keyboards/pom_keyboards/tnln95/tnln95.h:24]: the full layout has many overlap keys so it might be hard to use.
but I will try to arrange the keys.

Comment (NMHoang05) [reply keyboards/pom_keyboards/tnln95/tnln95.h]: I added `LAYOUT_all` to info.json

**Comment by noroadsleft** [comment]
@NMHoang05,

I feel like I should apologize for this trouble. I knew that the `info.json` was incorrect on #9788, but it was messed up in a way where I knew it'd be easier to fix it if I had an existing implementation in place. I was going to do that this morning, but you beat me to it. (QMK Configurator implementations are my area of expertise here.)

**Comment by NMHoang05** [comment]
@noroadsleft 
no worries 👍  
I was copy/paste the `info.json` and some parts weren't properly modified.

I have renamed the keymap.

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**